test/validation/api/pktio/pktio.c

```diff
@@ -3468,6 +3468,109 @@ static void pktio_test_recv_maxlen_set(void)
 	packet_len = PKT_LEN_NORMAL;
 }
 
+static int pktio_check_pktout_aging_tmo(void)
+{
+	odp_pktio_param_t pktio_param;
+	odp_pktio_capability_t capa;
+	odp_pktio_t pktio;
+	int idx = 0;
```

Comment:
This variable is not needed. Other pktio_check functions use iface_name[0] directly.